packages/react-aria-components/docs/Select.mdx

```diff
@@ -995,10 +995,38 @@ A [Button](Button.html) can be targeted with the `.react-aria-Button` CSS select
 
 ### SelectValue
 
-A `SelectValue` can be targed with the `.react-aria-SelectValue` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
+A `SelectValue` can be targeted with the `.react-aria-SelectValue` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
 
 <StateTable properties={docs.exports.SelectValueRenderProps.properties} />
 
+Select passes the rendered children of the selected item in the render props of the `SelectValue` as well as if the placeholder should be showing. You can use this to customize the value displayed in the Select at all times.
```

Comment:
This should go in its own section. It's not really related to "styling" and doesn't fit with the other sections there, so I think it should be a top-level section after "Controlled open state" before "Props".